internal/pkg/numalign/numalign.go

```diff
@@ -0,0 +1,147 @@
+/*
```

Comment:
I want to be sure I understand.
The project is nothing more than a collection of debugging tools, each of them will be in their own internal/pkg/dir and will end up in the the debug-tools image?
This image in turn will be used by a container thaw will be injected in the pod we want to debug?

Reply:
Or the image will be used by a privileged container on the host?

Reply:
I think your initial question deserves a better answer than one I can provide in the comments, so: https://github.com/openshift-kni/debug-tools/pull/4

regarding `pkg` vs `internal/pkg`, I keep in the latter code which is not (and in this case hardly will be) ready for public consumption: https://dave.cheney.net/2019/10/06/use-internal-packages-to-reduce-your-public-api-surface

Reply:
Got it, moving to #4